tools/cldr-apps/src/main/webapp/js/special/users.js

```diff
@@ -1,204 +1,202 @@
-/**
- * Special module that shows a users page. 

```

Comment:
All the whitespace changed in this file, but the only substantial difference is contextPath changed to cldrStatus.getContextPath()